modules/class-simplechart-save.php

```diff
@@ -96,6 +97,14 @@ protected function _do_save_post( $post ) {
 			}
 		}
 
+		// save subtitle separately since we want to be able to save empty strings
+		if ( isset( $_POST['save-chartSubtitle'] ) ) {
+			if ( 'false' !== $_POST['save-chartSubtitle'] ) {
```

Comment:
does this edge case with `'false'` apply to any of the other similar fields here?